metricbeat/module/kubernetes/util/kubernetes.go

```diff
@@ -42,9 +42,9 @@ import (
 type kubernetesConfig struct {
 	KubeConfig        string                       `config:"kube_config"`
 	KubeClientOptions kubernetes.KubeClientOptions `config:"kube_client_options"`
-
-	Node       string        `config:"node"`
-	SyncPeriod time.Duration `config:"sync_period"`
+	KubeAdm           bool                         `config:"use_kubeadm"`
```

Comment:
I don't think it is needed here as long as it is in https://github.com/elastic/elastic-agent-autodiscover/blob/a1dcbcec6e6b14f4a826d3a07c2eb93d9412907f/kubernetes/metadata/config.go#L27